### ISSUE TITLE
Django 1.11 is a dependency of django-jet 1.0.6

### ISSUE THREAD

**Issue by luca120289** [geex-arts/django-jet#214]
Hi,
today I tried to upgrade django-jet to the lastest version and when I try pip want to install also the last version of django.

Looking in the egg.info of django-jet 1.0.6 I see that, in the requirements, Django is a dependency.
It must be Django>=some.version (I don't know which is the minium required version of django). If the minimum version is not specified pip try to install the lastest version of the package.

I'm using Django 1.10.7 and (at the moment) I can't upgrade to the latest version.

Thanks

**Comment by SalahAdDin** [comment]
Yes, is a big problem, for now is better don't update `django` because incompatibility problems. @f1nality Can you change this?

**Comment by Ismael-VC** [comment]
Please come to the django-jet Discord server so we can organize if you like:

  *   https://discord.gg/3yr5yeH

Welcome! 😄

**Comment by SalahAdDin** [comment]
I will solve this problem in my fork, test it @luca120289
